modules/axios/plugin.js

```diff
@@ -118,6 +136,23 @@ export default (ctx) => {
   });
   <% } %>
 
+  <% if(options.debug) { %>
+  axios.interceptors.request.use(config => {
+    debug('[@nuxtjs/axios] Request:', config)
```

Comment:
Well actually, I used [axios] at first but then I thought that we want the user to know which module generates this log message. Say someone forgot he enabled this debug option, he could be spending a lot of time looking at why the axios package is logging al that info. So using `@nuxtjs/axios` makes it immeditaly where he should look